vowpalwabbit/packages.config

```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="boost" version="1.70.0.0" targetFramework="native" />
-  <package id="boost_program_options-vc141" version="1.70.0.0" targetFramework="native" />
-  <package id="boost_unit_test_framework-vc141" version="1.70.0.0" targetFramework="native" />
+  <package id="boost" version="1.71.0.0" targetFramework="native" />
```

Comment:
what is prompting this update, is it required by 3.1

Reply:
It came with an upgrade to v142 MSVC toolset (VS2019): this package https://www.nuget.org/packages/boost_unit_test_framework-vc142/1.71.0 has only 1.71 version.